Chameleon.iOS/Chameleon.iOS.csproj

```diff
@@ -1025,6 +1025,11 @@
     <PackageReference Include="Xamarin.Plugin.FilePicker">
       <Version>2.1.36-beta</Version>
     </PackageReference>
+    <PackageReference Include="StyleCop.Analyzers">
```

Comment:
Deze package kan toch in de Directory.build.props staan?

Reply:
Bedoel je de inhoud van die package kopiëren naar de directory. waarin de build.props staat?